docs/resources/tunnel.md

```diff
@@ -42,5 +42,5 @@ resource "cloudflare_tunnel" "example" {
 Import is supported using the following syntax:
 
 ```shell
-$ terraform import cloudflare_argo_tunnel.example <account_id>/<tunnel_id>
+$ terraform import cloudflare_tunnel.example <account_id>/<tunnel_id>
```

Comment:
you'll want to update this file instead https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/examples/resources/cloudflare_tunnel/import.sh#LL1C2-L1C2